### PR TITLE
Hotfix/db entities

### PR DESCRIPTION
**Pull request by dimacellist**
Added Magic Methods __toString to avoid Exception errors

### CODE REVIEW

**Comment by Kristute** [comment]
Looks good :+1: just for the commit messages check this https://chris.beams.io/posts/git-commit/
I got same remark from mentor :smile:

File: src/DataFixtures/AppFixtures 2.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+            $academy->setAcademyName('Super Academy '.$i);
+            $academy->setAcademyEmail('academy@email.com');
+            $academy->setAcademyUrl('www.academy.com');
+            $academy->setAcademyLogo('www.google.com/image.jpg');
+            $manager->persist($academy);
+        }
+        $manager->flush();
+    }
+}
```
Comment (Kristute) [inline src/DataFixtures/AppFixtures 2.php]: Is this really needed? same file in master :thinking:

Comment (dimacellist) [reply src/DataFixtures/AppFixtures 2.php]: Creating fixtures by the book:
https://symfony.com/doc/master/bundles/DoctrineFixturesBundle/index.html

Comment (dimacellist) [reply src/DataFixtures/AppFixtures 2.php]: Palauk,Fixtures 2??? Tuoj pachekinsiu

**Pull request merged**